thetanuts/thetanuts/config.py

```diff
@@ -45,11 +45,11 @@ def create_offer(
         if chain_id == Chains.MATIC:
             w3.middleware_onion.inject(geth_poa_middleware, layer=0)
         vaultContract = w3.eth.contract(
-            w3.toChecksumAddress(oToken),
+            w3.to_checksum_address(oToken),
             abi=get_abi("Thetanuts_Vault"),
         )
 
-        nonce = w3.eth.get_transaction_count(public_key)
+        nonce = w3.eth.get_transaction_count(web3.Web3.to_checksum_address(public_key))
```

Comment:
Argument 1 to "get_transaction_count" of "Eth" has incompatible type "str"; expected "Union[Address, ChecksumAddress, ENS]"

Reply:
Can we use `w3.to_checksum_address()` instead of `web3.Web3.to_checksum_address()`?

Reply:
Yes! Modified